### PR TITLE
feat:Added authentication

### PR DESCRIPTION
**Pull request by tushargoyal812**
social-auth

### CODE REVIEW

File: my-tweet/src/components/Login-modal.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+          onClose={onClose}
+        >
+          <ModalOverlay />
+          <ModalContent>
+            <ModalHeader>Login</ModalHeader>
+            <ModalCloseButton />
+            <ModalBody pb={6}>
+              <FormControl>
+                <FormLabel>User name</FormLabel>
+                <Input onChange={(e)=>setLogin({...login,username:e.target.value})} type="text" ref={initialRef} placeholder='User name' />
```
Comment (deepak29-git) [inline my-tweet/src/components/Login-modal.js:40]: you can make a function so that you would not repeat code twice

**Review by deepak29-git — APPROVED** [review]
variable name is good

**Review by sharmayatendra — APPROVED** [review]
code looks clean 
design is consistent
file structure is good

File: my-tweet/src/components/Login-modal.js (hunk 1 above)
Comment (tushargoyal812) [reply my-tweet/src/components/Login-modal.js:40]: ok will do it

**Pull request merged**